client/src/components/AboutPage/AboutPage.tsx

```diff
@@ -1,9 +1,15 @@
 import React, { FunctionComponent } from 'react';
 
+// Components
+import Header from 'components/Header/Header';
+
 const AboutPage: FunctionComponent = () => (
-  <main>
-    <h1>About Page</h1>
-  </main>
+  <>
+    <Header />
```

Comment:
please create layout component with using `<Outlet/>` to reduce number of `Header` component renders